ui/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "license": "MIT",
   "engines": {
-    "node": ">=16.0.0 <17.0.0"
```

Comment:
We need this to stay on CI - or we should deliberately bump the node version

Reply:
I'll update this :+1: